Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -22,7 +22,7 @@
 
 
 # ------------------------------------------------------------------------------
-def loadPatientByUID(patientUID):
+def loadPatientByUID(patientUID, messages=None, progressCallback=None):
```

Comment:
```suggestion
def loadPatientByUID(patientUID, messages: Optional[list[str]]=None, progressCallback=None):
```

You  also need to add the following at the top:

```
from typing import Optional
```